characters/watermelon.py

```diff
@@ -101,9 +101,24 @@ def fire_seed(self, model, seed_velocity):
             if self.health > 0:
                 self.health -= 10
 
+            state.offset = self.screen_shake(2, 3)
```

Comment:
maybe check that a bullet was fired before shaking? right now you can hit spacebar on the "game over" screen and it'll shake lol

Reply:
Turns out you can still fire bullets on game over. I can look at adding some additional checks to prevent input on game over.